common/lib/xmodule/xmodule/modulestore/mongo/base.py

```diff
@@ -151,12 +151,28 @@ def has(self, key):
         else:
             return False
 
+    def __repr__(self):
+        return "MongoKeyValueStore{!r}<{!r}, {!r}>".format(
+            (self._data, self._parent, self._children, self._metadata),
+            self._fields,
+            self.inherited_settings
+        )
+
```

Comment:
https://docs.python.org/2/reference/datamodel.html#object.__repr__

Basically, I try to get `__repr__` as close as possible to the python expression to re-create the object.